### ISSUE TITLE
Definition of "vocabulary", Section 1.2

### ISSUE THREAD

**Issue by klensin** [w3c/charmod-norm#60]
The last sentence of this definition reads "ECMAScript restricts the range of characters that can appear at the start or in the body of an identifier or variable name (while different rules apply to the values of, say, string literals)."  
That seems unnecessarily hard to follow.  It might be improved by substituting:
"ECMAScript restricts the range of characters that can appear at the start or in the body of an identifier or variable name.  It applies different rules for other cases, e.g., to the values of string literals."

**Comment by aphillips** [comment]
Taken almost verbatim:

> Another example would be ECMAScript, which restricts the range of characters that can 
> appear at the start or in the body of an identifier or variable name. It applies different rules 
> for other cases, such as to the values of string literals.

**Comment by aphillips** [comment]
Closing. Reopen if unsatisfied.
